src/Angor/Avalonia/Angor.Wallet/Domain/WalletDomainService.cs

```diff
@@ -0,0 +1,12 @@
+using CSharpFunctionalExtensions;
+
+namespace Angor.Wallet.Domain;
+
+public static class WalletDomainService
```

Comment:
This is a bit confusing to me, are the transactions part of the wallet? where is the data coming from?

Reply:
You're right - I initially created a WalletDomainService thinking we needed domain logic for balance calculation, but I realized it wasn't necessary because:

1. Transactions aren't part of the Wallet entity:
   - They live in the blockchain (our source of truth)
   - The balance is just a derived value from these transactions

2. The data flow is straightforward:
   - Transactions come from our blockchain node
   - Balance is a simple sum of unspent outputs
   - No complex domain rules are involved

I'll remove the WalletDomainService since it's adding unnecessary complexity. The balance calculation can be handled directly in the infrastructure layer where we interact with the blockchain node.